### PR TITLE
Improved 'bad' example for the Strings section

### PR DESCRIPTION
**Pull request by igorradovanov**
## Description

Bad and good examples are the same. I've added the double quotes to bad example as intended to show the difference between what is bad and what is a good example for using strings in the coding standards for the Gutenberg editor.

## How has this been tested?

Verified that the patch doesn't break the flow of the page.

## Types of changes

Improved the example code for the 'Bad' example.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: docs/contributors/coding-guidelines.md [hunk 1]
```diff
@@ -174,7 +174,7 @@ In general, avoid backslash-escaping quotes:
 
 ```js
```
Comment (ZebulanStanphill) [inline docs/contributors/coding-guidelines.md:175]: It looks like this got changed automatically by Prettier, so next time someone edits this file and saves, the problem might be brought back. Ironically, our tooling is ruining the bad example by turning it into a good example. I think adding a `<!-- prettier-ignore -->` before the code block will prevent that from happening again.

```suggestion
<!-- prettier-ignore -->
```js
```

See also: https://prettier.io/docs/en/ignore.html

**Review by ZebulanStanphill — CHANGES_REQUESTED** [review]

File: docs/contributors/coding-guidelines.md (hunk 1 above)
Comment (igorradovanov) [reply docs/contributors/coding-guidelines.md:175]: Thank you. I will check if the prettier-ignore tag is the proper solution and submit another patch.

Comment (gziolo) [reply docs/contributors/coding-guidelines.md:175]: I have just tested it, and I can confirm that @ZebulanStanphill is right 💯

**Review by gziolo — APPROVED** [review]

**Comment by gziolo** [comment]
I added a follow-up commit where I marked 2 more JS code sections to be ignored by Prettier and tested with a Prettier call. It works as advertised by @ZebulanStanphill. See: https://github.com/WordPress/gutenberg/commit/2d529f5ebfaade01b040c198312807603ac81e4a.

**Pull request merged**